### DISCUSSION TITLE
Welcome to frida-ios-hook Discussions!

### DISCUSSION THREAD

**Discussion by yazgx97** [noobpk/frida-ios-hook#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Shouzaizai** [comment]
可以编写一个关于运行在window的详细教程吗？ 很需要 谢谢！

**Comment by Shouzaizai** [comment]
Hello! Can you write a tutorial about running in window? Thank you!
